content/blog/2023/contain-root.md

```diff
@@ -0,0 +1,517 @@
+---
+title: Can We Query the Root Container?
+sub: The complexities of containment, overflow, and 'propagation'
+author: miriam
+date: 2023-07-03
+tags:
+  - Article
+  - CSS
+  - Container Queries
+image:
+  src: blog/2023/pwc23.png
+summary: |
+  I spoke about Container Queries
+  at both Smashing Conference (San Francisco)
+  and CSS Day (Amsterdam) --
+  where I recommended setting up a root container
+  to replace most media queries.
+  Since then, Temani Afif
```

Comment:
Link here to Temani Afif since it's the first mention?

Reply:
I don't think we allow links in summaries? Or do we strip them out automatically in the list view?

Reply:
I think we allow them, but strip them out for the list view. Try it and see?